src/libs/metadata_view.c

```diff
@@ -994,18 +994,21 @@ static void _metadata_view_update_values(dt_lib_module_t *self)
 
 static void _jump_to()
 {
-  int32_t imgid = dt_control_get_mouse_over_id();
-  if(imgid == -1)
+  dt_imgid_t imgid = dt_control_get_mouse_over_id();
```

Comment:
isnt this const here

Reply:
No because modified a bit later (already reported :).